cmd/cloudStorage/main.go

```diff
@@ -58,6 +63,9 @@ func main() {
 	// initialize the service
 	service := storage.New(s3, keys, publisher.NewNullPublisher(context.Background()), logger.With().Str("component", "service/storage").Logger())
 
+	// initialize authorizer
+	auth := authorizer.New("https://cloudAuth/auth/validate", logger.With().Str("component", "service/authorizer").Logger())
```

Comment:
Just pass `logger`. Components should add `component` field to logger by themselves.

Reply:
yeah, now I recall @dominikznidar mentioning that will sweep the code for those whenever I'll finish all the other changes.